packages/SketchMorph2-Core.package/M2DrawingComponent.class/instance/dragCircleOn.at..st

```diff
@@ -1,7 +1,7 @@
 drawing
 dragCircleOn: aForm at: aRectangle
 	| temporaryPen diameter radius center |
-	diameter := aRectangle width min: aRectangle height . 
+	diameter := aRectangle width min: aRectangle height .
```

Comment:
```suggestion
	diameter := aRectangle width min: aRectangle height.
```

Reply:
I don't know if we have to add this to the auto refactoring since this seems to be a very rare mistake, but it's a mistake nonetheless.

Reply:
I don't think this would be possible, considering the fact that `.` is also the delimiter for collections, that may well use a space before a full stop at a line end. Also considering that this mistake is very, very rare, I don't see the point in trying to add a rule for this.